src/time/wasm.rs

```diff
@@ -0,0 +1,28 @@
+#[cfg(target_family = "wasm")]
+pub struct Instant {
+    start: f64,
+}
+
+#[cfg(target_family = "wasm")]
+impl Instant {
+    pub fn now() -> Self {
+        let window = web_sys::window().unwrap();
```

Comment:
looks to me you did not add the web_sys dep to the project ? :)

I know this is mainly to showcase how to debug it, but if the patch is not complete it will be hard for us to integrate :)